dev-packages/browser-integration-tests/suites/integrations/featureFlags/statsig/basic/test.ts

```diff
@@ -0,0 +1,47 @@
+import { expect } from '@playwright/test';
+
+import { sentryTest } from '../../../../../utils/fixtures';
+
+import { envelopeRequestParser, shouldSkipFeatureFlagsTest, waitForErrorRequest } from '../../../../../utils/helpers';
+
+const FLAG_BUFFER_SIZE = 100; // Corresponds to constant in featureFlags.ts, in browser utils.
```

Comment:
we might want to move this into a module in like `dev-packages/browser-integration-tests/suites/integrations/featureFlags` or something